lib/core/src/lib/viewer/components/viewer-render/viewer-render.component.ts

```diff
@@ -185,16 +185,22 @@ export class ViewerRenderComponent implements OnChanges, OnInit {
     }
 
     ngOnChanges() {
-        this.isContentReady = false;
-        this.isLoading = !this.blobFile && !this.urlFile;
+        this.checkIsLoaded();
 
         if (this.blobFile) {
             this.setUpBlobData();
+            this.checkIsLoaded();
         } else if (this.urlFile) {
             this.setUpUrlFile();
+            this.checkIsLoaded();
         }
```

Comment:
Why do we need to invoke `checkIsLoaded()` twice on changes in case of `blobFile` and `urlFile`, is this correct?